cli/dts/lib.deno.window.d.ts

```diff
@@ -83,6 +85,8 @@ declare var onunhandledrejection:
 declare var localStorage: Storage;
 /** @category Web Storage API */
 declare var sessionStorage: Storage;
+/** @category Cache API */
+declare var caches: CacheStorage;
```

Comment:
Just to double check, it's on purpose that caches are only available in the main worker and not web workers?

Reply:
The API does exist for workers at runtime, so just a typings issue I think.

Reply:
updated `lib.deno.worker.d.ts`